src/components/ModalBackupWords.js

```diff
@@ -51,33 +51,37 @@ class ModalBackupWords extends React.Component {
       passwordSuccess: false,
       showValidation: false,
       passwordFormValidated: false,
-      wordsValidation: [],
-      chosenWords: [],
-    }
+      validationSteps: [],
```

Comment:
I would be good to have a description of this state in the comment before `this.state = `. As far as I understand it's an array of objects, so it should have each key of the object the its meaning.

Besides that, you should remove `wordsValidation` and  `chosenWords` from the comment because they were removed

Reply:
Done